docs/UserGuide.md

```diff
@@ -256,7 +256,7 @@ Examples:
 
 Undoes the previous command that the user has entered, which has changed the data within CCACommander.
 
-Format: `redo`
+Format: `undo`
```

Comment:
LGTM! But I think it will be good for us to highlight to the users which specific commands can be undone (commands that change data within the JSON file).

Reply:
Agreed, but I think we can add this in a separate PR